test/e2e/framework.go

```diff
@@ -751,10 +746,26 @@ func (data *TestData) deployFlowVisibilityClickHouse() (string, error) {
 	return chSvc.Spec.ClusterIP, nil
 }
 
+func (data *TestData) deleteFlowVisibility() error {
+	startTime := time.Now()
+	defer func() {
+		log.Infof("Deleting K8s resources created by flow visibility YAML took %v", time.Since(startTime))
+	}()
+	rc, _, _, err := data.provider.RunCommandOnNode(controlPlaneNodeName(), fmt.Sprintf("kubectl delete -f %s", flowVisibilityYML))
```

Comment:
@heanlan could you help me understand what was the issue? I assume it was affected by the change made to `DeleteNamespace`that it no longers wait for namespace deletion, did it affect deletion of ClickHouseOperator?

Reply:
That change affected other tests as well, @gran-vmv had PR #3835 which makes `DeleteNamespace` wait for deletion complete if timeout >=0, would it resolve the cleanup issue of ClickHouseOperator? If yes, I would merge this PR first as long as it fixes `checkRecordsForFlowsClickHouse` and we could leave cleanup issue to #3835.

Reply:
We observed the "test timed out after 10m0s" issue. And I found it is because the test stucks in the deletion of clickhouse operator. 

I didn't realized the DeleteNamespace no longer wait for namespace deletion. That should be the direct cause of the test timeout issue. clickhouse operator need to be deleted AFTER we delete ClickHouseInstallation CR created in flow-visibility namespace.

The fix I am proposing in commit#2 is - instead of deleting flow-visbility namespace, we should delete -f flow-visibility.yml. Deleting namespace won't delete the pv and the storageclass. We believe this approach performs a complete deletion. That is also what we're suggesting in the [documentation](https://github.com/antrea-io/antrea/blob/main/docs/network-flow-visibility.md#deployment-steps-2). Otherwise, we may run into issue if we only delete flow-visibility namespace, and re-apply flow-visibility.yml. In the perspective of e2e test, we'll run into issue if we re-run the test on the same node.

Reply:
@tnqn sorry I didn't see your second comment while typing the reply.

Yes, the fix in #3835 is good enough to fix the timeout issue. I was just wondering do you think it is better if we change from deleting namespace to deleting by file.

Reply:
Thanks for the explanation. Your current approach makes sense to me and it makes the test code more symmetric